Documentation/ceph-csi-drivers.md

```diff
@@ -108,105 +121,110 @@ Once the plugin is successfully deployed, test it by running the following examp
 
-Run `ceph auth ls` in your rook ceph operator pod, to encode the key of your
-admin/user run `echo -n KEY|base64`
-and replace `BASE64-ENCODED-PASSWORD` by your encoded key.
+kubernetes_key=$(pod=$(kubectl get pod -n rook-ceph -l app=rook-ceph-operator -o jsonpath="{.items[0].metadata.name}"); kubectl exec -ti -n rook-ceph ${pod} -- bash -c "ceph auth get-key client.kubernetes -c /var/lib/rook/rook-ceph/rook-ceph.config | base64")
+```
 
 ```bash
-kubectl exec -ti -n rook-ceph rook-ceph-operator-6c49994c4f-pwqcx /bin/sh
-sh-4.2# ceph auth ls
-installed auth entries:
+cat << EOF | kubectl create -f -
+apiVersion: v1
+kind: Secret
+metadata:
+  name: csi-rbd-secret
+  namespace: default
+data:
+  # Key value corresponds to a user name defined in Ceph cluster
+  admin: ${admin_key}
+  # Key value corresponds to a user name defined in Ceph cluster
+  kubernetes: ${kubernetes_key}
+  # if monValueFromSecret is set to "monitors", uncomment the
+  # following and set the mon there
+  #monitors: BASE64-ENCODED-Comma-Delimited-Mons
+EOF
+```
 
-osd.0
-	key: AQA3pa1cN/fODBAAc/jIm5IQDClm+dmekSmSlg==
-	caps: [mgr] allow profile osd
-	caps: [mon] allow profile osd
-	caps: [osd] allow *
-osd.1
-	key: AQBXpa1cTjuYNRAAkohlInoYAa6A3odTRDhnAg==
-	caps: [mgr] allow profile osd
-	caps: [mon] allow profile osd
-	caps: [osd] allow *
-osd.2
-	key: AQB4pa1cvJidLRAALZyAtuOwArO8JZfy7Y5pFg==
-	caps: [mgr] allow profile osd
-	caps: [mon] allow profile osd
-	caps: [osd] allow *
-osd.3
-	key: AQCcpa1cFFQRHRAALBYhqO3m0FRA9pxTOFT2eQ==
-	caps: [mgr] allow profile osd
-	caps: [mon] allow profile osd
-	caps: [osd] allow *
-client.admin
-	key: AQD0pK1cqcBDCBAAdXNXfgAambPz5qWpsq0Mmw==
-	auid: 0
-	caps: [mds] allow *
-	caps: [mgr] allow *
-	caps: [mon] allow *
-	caps: [osd] allow *
-client.bootstrap-mds
-	key: AQD6pK1crJyZCxAA1UTGwtyFv3YYFcBmhWHyoQ==
-	caps: [mon] allow profile bootstrap-mds
-client.bootstrap-mgr
-	key: AQD6pK1c2KaZCxAATWi/I3i0/XEesSipy/HeIA==
-	caps: [mon] allow profile bootstrap-mgr
-client.bootstrap-osd
-	key: AQD6pK1cwa+ZCxAA7XKXRyLQpaHZ+lRXeUk8xQ==
-	caps: [mon] allow profile bootstrap-osd
-client.bootstrap-rbd
-	key: AQD6pK1cULmZCxAA4++Ch/iRKa52297/rbHP+w==
-	caps: [mon] allow profile bootstrap-rbd
-client.bootstrap-rgw
-	key: AQD6pK1cbMKZCxAAGKj5HaMoEl41LHqEafcfPA==
-	caps: [mon] allow profile bootstrap-rgw
-mgr.a
-	key: AQAZpa1chl+DAhAAYyolLBrkht+0sH0HljkFIg==
-	caps: [mds] allow *
-	caps: [mon] allow *
-	caps: [osd] allow *
+## Create RBD StorageClass
 
-#encode admin/user key
-sh-4.2# echo -n AQD0pK1cqcBDCBAAdXNXfgAambPz5qWpsq0Mmw==|base64
-QVFEMHBLMWNxY0JEQ0JBQWRYTlhmZ0FhbWJQejVxV3BzcTBNbXc9PQ==
-#or
-sh-4.2# ceph auth get-key client.admin|base64
-QVFEMHBLMWNxY0JEQ0JBQWRYTlhmZ0FhbWJQejVxV3BzcTBNbXc9PQ==
-```
+Create a ``storage-class.yaml`` file, be sure to update the ``monitors`` entry to reflect the IP of your
```

Comment:
Just curious, why use double ticks? Looks like it renders the same as single ticks.

Reply:
My vim plugin automatically makes double ticks, and because they do in fact render the same I didn't see any reason that it would be problematic.  I guess if they are annoying I can remove them though.

Reply:
that's fine with double ticks, no need to change